### PR TITLE
adds common-issues, starting with wsl2 forwarding

### PR DESCRIPTION
**Pull request by pablohildo**
Page structure is slightly based on [Microsoft's Azure](https://docs.microsoft.com/pt-br/azure/cyclecloud/common-issues/execute_command?view=cyclecloud-7).

Can be understood as a tree:

- Common Issues
    - Issue name
        - Observed Issue (Issue Description)
        - Resolution (Simple, straightforward solution)
        - Additional Information (Any remaining info, references, more context)

If done accordingly to this proposal, the repeating part shall be "Issue name" and its children.

### CODE REVIEW

**Comment by kevinoliveira** [comment]
would be nice to have a reference to this problem on the [getting started section ](https://sdkgen.github.io/getting-started-first-nodejs-server)

**Comment by lbguilherme** [comment]
I think this is good as is. "Getting started" page should be as small as possible. Having the "common issues" on the menu should be enough.

**Review by lbguilherme — APPROVED** [review]

File: common-issues.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+route: /common-issues
+---
+
+# Common Issues
+
+## Requests aren't received when using WSL2
+
+### Observed Issue
+
+Although `new SdkgenHttpServer.listen(port)` is being called and outputs `Listening on :::PORT` or similar to console, no connection can be stablished. Hence, no requests are received.
```
Comment (dgadelha) [inline common-issues.mdx]:
```suggestion
Although `new SdkgenHttpServer.listen(port)` is being called and outputs `Listening on :::PORT` or similar to console, no connection can be established. Hence, no requests are received.
```

File: common-issues.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### Resolution
+
+WSL2 ports aren't always forwarded. Two solutions are appliable:
+- Use a default forwarded port (`3000` is most likely one of those);
+- Add a new port forwarding rule that includes the port you're using. Remember to also add this rule to Firewall Rules, otherwise it's probably going to be blocked.
```
Comment (dgadelha) [inline common-issues.mdx]: It'd be better to add a link do a tutorial on this solution

File: common-issues.mdx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+### Additional Information
+
+This is a known issue with WSL2 as seen on it's [official repository](https://github.com/microsoft/WSL/issues/4150). The reason this happens is WSL2 uses a completely new network system architecture, "changing from the default bridged network adapter to a hyper-v virtual network adapter". You can find more Information [here](https://github.com/microsoft/WSL/issues/4150#issuecomment-504209723).
```
Comment (dgadelha) [inline common-issues.mdx]:
```suggestion
This is a known issue with WSL2 as seen on its [official repository](https://github.com/microsoft/WSL/issues/4150). The reason this happens is WSL2 uses a completely new network system architecture, "changing from the default bridged network adapter to a Hyper-V virtual network adapter". You can find more information [here](https://github.com/microsoft/WSL/issues/4150#issuecomment-504209723).
```

**Review by dgadelha — CHANGES_REQUESTED** [review]

File: common-issues.mdx (hunk 3 above)
Comment (pablohildo) [reply common-issues.mdx]: Oops, right over my head. Fixing this right now, but I'm slightly unsure if we should change "hyper-v" specifically. Although I do agree with you, I'm directly citing the comment I linked immediately after (the whole reason the quotation marks are there).
I don't see any problems on using it like you suggested, but I think it's worth looking at it.

Comment (pablohildo) [reply common-issues.mdx]: Made the suggested changes with the above exception. 
Will take action after your feedback.

**Pull request closed without merging**